services/buildbot/master/twisted_factories.py

```diff
@@ -689,8 +689,12 @@ class TwistedBdistFactory(TwistedBaseFactory):
     def __init__(self, source, uncleanWarnings, arch, pyVersion):
         python = self.python(pyVersion)
         TwistedBaseFactory.__init__(self, python, source, uncleanWarnings)
-        self.addStep(
-            LearnVersion, python=python, package='twisted', workdir='Twisted')
+        learnVersion = LearnVersion(
+            python=python,
+            package='twisted',
+            workdir='Twisted',
+            )
+        self.addStep(learnVersion)
```

Comment:
Yeah, I think 0.8.7 dropped support for the `addStep(stepClass, *args)` form, in favor of just passing a constructed step (which then magically acts as a factory.